documentation/docker/compose/docker-compose.yml

```diff
@@ -15,7 +15,7 @@ services:
    helloservice:
      image: kvalitetsit/kithugs:latest
      environment:
-      - jdbc_url=jdbc:mariadb://mariadb:3306/hellodb
+      - jdbc_url=jdbc:mysql://mariadb:3306/hellodb #used to be "jdbc_url=jdbc:mariadb://mariadb:3306/hellodb", was changed to fix crashes on startup
```

Comment:
Hvis du bygger projektet lokalt, og dermed får et docker image med tagget latest, så skal det være mariadb driveren der bruges. Hvis det virker med mysql driveren, så vil det være et meget gammelt image du har fat i, da vi ikke har brugt mysql driveren i lang tid.

Reply:
Jeg har implementeret alt det, du foreslog. Tak for den grundige feedback, det var lærerigt :+1: